yarn-error.log

```diff
@@ -0,0 +1,6847 @@
+Arguments: 
```

Comment:
Remove this file or add it to `.gitignore`

Reply:
Done in [3b1e9af](https://github.com/pagopa/hub-spid-login-ms/pull/54/commits/3b1e9af44a18828a6bc032b010cb0daddb71d562)